cypress/integration/finish-button-test.js

```diff
@@ -0,0 +1,90 @@
+
+describe("determine finish button and pop up indicator", function () {
+  const email = Cypress.env("email");
+  const password = Cypress.env("password");
+  const baseUrl = Cypress.env("baseUrl");
+  const viewportWidth = Cypress.env("viewportWidth");
+  const viewportHeight = Cypress.env("viewportHeight");
+  const value = 1;
```

Comment:
nitpicking here, but be more specific here - image if you're a junior dev reading this and trying to understand what these values are for.

Maybe mrlValue?

Reply:
mrlValue makes sense!